salt/cni/cilium-ds.yaml.jinja

```diff
@@ -160,8 +160,9 @@ spec:
       tolerations:
         # Allow the pod to run on the master.  This is required for
         # the master to communicate with pods.
-        - operator: Exists
-          effect: NoSchedule
+      - key: node-role.kubernetes.io/master
```

Comment:
This should go to the previous commit? d74c65b5fd30fcd8c747b4b3f94545ac8766143

Reply:
squashed into one commit